src/Tizen.Uix.Tts/Tizen.Uix.Tts/TtsClient.cs

```diff
@@ -676,7 +691,7 @@ public State CurrentState
         /// <exception cref="OutOfMemoryException">This exception can be due to out Of memory.</exception>
         /// <exception cref="NotSupportedException">This exception can be due to TTS not supported.</exception>
         /// <pre>
-        /// The State should be created.
+        /// The state must be created.
```

Comment:
it's great to improve this description, I would suggest rephrasing it to not confuse with creating a state - as if the "created" was a part of verb, but that "The Client must be in the `created` state" or "in the state `created`" or "The Client state must be `created`" or "The state must be \<see cref="State.Created"/\>",
same for other state prerequisites